modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/dnd_table/Cell.js

```diff
@@ -131,7 +131,7 @@ function Cell({
 
 Cell.defaultProps = {
 	heading: false,
-	resizable: false,
+	resizable: true,
```

Comment:
I don't think this is the right solution for the problem. What `resizable` allows is for user to _manually_ change column width. It'll still be broken first time any user opens it, and it may not be obvious that resize option is available. Even if it was obvious, it is a needles pain point for end users. Another detail is that widths are currently persisted, but in an ugly way that we want to remove ones custom views are implemented. After that change, width will reset with each page refresh.  

A better solution would be to set styles to automatically compensate increased width.

Reply:
Thank you for the information!

I have been looking into setting styles to compensate for the increased width and found that having the `width: auto` for `dnd-td item-actions`, which are the last cells will work for the bottom but the top cell will not extend the length of the table.

Can I get some guidance on how I could make the cells function correctly? 

![Screenshot from 2024-03-05 16-03-03](https://github.com/liferay-frontend/liferay-portal/assets/31672551/32702c03-5beb-4b66-9fde-27b3b1ae6b9f)

Reply:
Hello @markocikos 

Do you have time to provide guidance on this issue, or point me to someone else?

Truly appreciate all the help on this!

Reply:
[Relevant discussion on Slack](https://liferay.slack.com/archives/C3JBR21HA/p1710231619681329)

Reply:
Hey @markocikos thanks for your comment, what about if we leave everything at it is, and center the caret if the button is too long? <img width="1266" alt="Screenshot 2024-03-12 at 15 44 17" src="https://github.com/liferay-frontend/liferay-portal/assets/98747059/a255e547-9413-42b7-8a74-2ee42328b981">

Reply:
I think we should stick to achieve this kind of visualization

![image](https://github.com/liferay-frontend/liferay-portal/assets/4663897/9b92350e-09df-4306-ba5c-844152c2506c)

where the button gets as width as needed. When the admin user renders the FDS will realize if is chosing too long labels.

Reply:
After understanding better the problem: The header width can't be adapted to the contents right now, so we make the column a little bit wider to adjust to most of cases.